pkg/monitoring/vmistats/collector_test.go

```diff
@@ -217,5 +221,66 @@ var _ = Describe("Utility functions", func() {
 			Expect(countMap[scheduling]).To(Equal(uint64(2)))
 			Expect(countMap[bogus]).To(Equal(uint64(0))) // intentionally bogus key
 		})
+
+		var instanceTypes = &vmisInstanceTypes{
+			instanceTypes: []*instancetypev1alpha2.VirtualMachineInstancetype{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:   "i-managed",
+						Labels: map[string]string{"instancetype.kubevirt.io/vendor": "kubevirt.io"},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "i-unmanaged",
+					},
+				},
+			},
+			clusterInstanceTypes: []*instancetypev1alpha2.VirtualMachineClusterInstancetype{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:   "ci-managed",
+						Labels: map[string]string{"instancetype.kubevirt.io/vendor": "kubevirt.io"},
+					},
+				},
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{"instancetype.kubevirt.io/vendor": "XPTO"},
+					},
+				},
+			},
+		}
+
+		DescribeTable("should show instance type value correctly", func(instanceTypeAnnotationKey string, instanceType string, expected string) {
+			annotations := map[string]string{}
+			if instanceType != "" {
+				annotations[instanceTypeAnnotationKey] = instanceType
+			}
+
+			vmis := []*k6tv1.VirtualMachineInstance{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:        "running",
+						Annotations: annotations,
+					},
+				},
+			}
+
+			countMap := makeVMICountMetricMap(vmis, instanceTypes)
+			Expect(countMap).NotTo(BeNil())
```

Comment:
This isn't required given the next line.

Reply:
removed